silkrpc/core/evm_executor.hpp

```diff
@@ -81,6 +85,7 @@ class EVMExecutor {
     boost::asio::thread_pool& workers_;
     state::RemoteState& remote_state_;
     WorldState state_;
+    std::unique_ptr<silkworm::consensus::IEngine> _consensus_engine;
```

Comment:
According to our name conventions, the leading underscore should be a trailing one (see also other member variables here): `consensus_engine_`

:smile:

Reply:
typo.